mutabazigakuba/Auto_Mart
# #166406387 Built and designed front-end for a user to mark his/her ad sold

#### What does the PR do?
- It's a front-end design for a user to click on to mark his/her car sale ad as sold
#### Description of Task to be completed?
- Have the link [mark as sold] the page update_price below visiable
../Auto_Mart/UI/pages/update_price.html
#### How should this be manually tested?
- After cloning the project into your PC, one can visit the link ../Auto_Mart/UI/pages/update_price.html
- then should be able to view `Mark As Sold` link below the car sale ad
#### What are the relevant pivotal tracker stories?
#166406387 [Pivotal Tracker Project](https://www.pivotaltracker.com/n/projects/2352055)
